.github/workflows/deny.yml

```diff
@@ -0,0 +1,32 @@
+name: Deny
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ["main"]
+    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
+  merge_group:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
```

Comment:
```suggestion
          - bans
          - licenses
          - sources
```

Aren't these individual checks?

Reply:
Literally copied from https://github.com/EmbarkStudios/cargo-deny-action?tab=readme-ov-file#recommended-pipeline-if-using-advisories-to-avoid-sudden-breakages

Reply:
I think what is there is correct, i.e., it is **one** run that checks **three** things.